sdk/keyvault/azure-security-keyvault/azure/security/keyvault/certificates/_client.py

```diff
@@ -0,0 +1,187 @@
+        # type: (Optional[bool], Mapping[str, Any]) -> Iterable[DeletedCertificate]
+        pass
+
+    def list_certificates(self, include_pending=None, **kwargs):
+        # type: (Optional[bool], Mapping[str, Any]) -> Iterable[CertificateBase]
+        pass
+
+    def list_certificate_versions(self, name, **kwargs):
+        # type: (str, Mapping[str, Any]) -> Iterable[CertificateBase]
+        pass
+
+    def create_contacts(self, contacts, **kwargs):
+        # type: (Iterable[Contact], Mapping[str, Any]) -> Iterable[Contact]
+        # TODO: rest api includes an id should it be returned?
+        pass
+
+    def list_contacts(self, **kwargs):
+        # type: (Mapping[str, Any]) -> Iterable[Contact]
+        pass
+
+    def delete_contacts(self, **kwargs):
+        # type: (Mapping[str, Any]) -> Iterable[Contact]
+        pass
+
+    def get_certificate_operation(self, name, **kwargs):
+        # type: (str, Mapping[str, Any]) -> CertificateOperation
+        pass
+
+    def delete_certificate_operation(self, name, **kwargs):
+        # type: (str, Mapping[str, Any]) -> CertificateOperation
+        pass
+
+    def cancel_certificate_operation(self, name, cancellation_requested, **kwargs):
+        # type: (str, bool, Mapping[str, Any]) -> CertificateOperation
+        pass
+
+    def get_pending_certificate_signing_request(self, name, **kwargs):
+        # type: (str, Mapping[str, Any]) -> str
+        pass
+
+    def get_issuer(self, name, **kwargs):
+        # type: (str, Mapping[str, Any]) -> Issuer
+        pass
+
+    def create_issuer(
+        self,
+        name,
+        provider,
+        account_id=None,
+        password=None,
+        organization_id=None,
+        admin_details=None,
+        enabled=None,
+        not_before=None,
+        expires=None,
+        tags=None,
+        **kwargs
+    ):
+        # type: (str, str, Optional[str], Optional[str], Optional[str], Optional[List[models.AdministratorDetails]], Optional[bool], Optional[datetime], Optional[datetime], Optional[Dict[str, str]], Mapping[str, Any]) -> Issuer
+        pass
+
+    def update_issuer(
+        self,
+        name,
+        provider=None,
+        account_id=None,
+        password=None,
+        organization_id=None,
+        first_name=None,
+        last_name=None,
+        email=None,
+        phone=None,
+        enabled=True,
+        not_before=None,
+        expires=None,
+        tags=None,
+        **kwargs
+    ):
+        # type: (str, Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[bool], Optional[datetime], Optional[datetime], Optional[Dict[str, str]], Mapping[str, Any]) -> Issuer
+        pass
+
+    def delete_issuer(self, name, **kwargs):
+        # type: (str, Mapping[str, Any]) -> Issuer
+        pass
+
+    def list_issuers(self, **kwargs):
+        # type: (Mapping[str, Any]) -> Iterable[IssuerBase]
```

Comment:
`Iterable` supports `__iter__`, `Iterator` supports `__iter__` and `__next__`. A list is `Iterable` but not an `Iterator`. A generator is both. If the return here will be a generator, `Iterator` is the right choice.